src/index.js

```diff
@@ -1,14 +1,65 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+let newTask;
+const arrowBtn = document.getElementById('arrow');
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+class Todolist {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
 
-  return element;
+  addtask(dataBase) {
+    dataBase.push(this);
+  }
 }
 
-document.body.appendChild(component());
\ No newline at end of file
+arrowBtn.addEventListener('click', () => {
+  const taskName = document.getElementById('input').value;
+  let dataBase = JSON.parse(localStorage.getItem('baseData'));
+  if (dataBase === null) {
+    dataBase = [];
+  }
+  const index = dataBase.length + 1;
+  newTask = new Todolist(taskName, false, index);
+  newTask.addtask(dataBase);
+  localStorage.setItem('baseData', JSON.stringify(dataBase));
+  document.location.reload();
+});
+
+function displaytask() {
+  const dataBase = JSON.parse(localStorage.getItem('baseData'));
+
+  const unsortedList = document.querySelector('.list');
+
+  for (let i = 0; i < dataBase.length; i += 1) {
+    const list = document.createElement('li');
+    list.id = i + 1;
+    unsortedList.appendChild(list);
+
+    const checkbox = document.createElement('input');
+    checkbox.classList.add('check');
+    checkbox.type = 'checkbox';
+    list.appendChild(checkbox);
+
+    const span = document.createElement('span');
+    span.classList.add('span');
+    span.textContent = dataBase[i].description;
+    list.appendChild(span);
+
+    const removeicon = document.createElement('button');
+    removeicon.classList.add('remove');
+    removeicon.innerHTML = '<i class="fa-solid fa-trash-can"></i>';
+    list.appendChild(removeicon);
+
+    removeicon.addEventListener('click', () => {
+      dataBase.splice(i, 1);
+      list.remove();
+
```

Comment:
**It is good that you have implemented the remove button but there is an issue with your delete function. How you can check that.**
- First create three todos
- Then delete all todos one by one from the top (**not from the bottom**)
- After deleting all todos refresh the page.

**You would see that there is one todo left which you have already deleted**

- [ ] Please fix this issue :+1:
- [ ] After removing a todo, the index of all todos should be reset. This means if you have 5 todos from index 1 to 5 then if you remove the first todo with index 1 then the second todo with index 2 will have index 1. Please implement this requirement :+1:

Reply:
indexes not fulfilled 🚫